docs/README.md

```diff
@@ -557,6 +557,35 @@ successText = "Thanks!"
 
 <!-- tabs:end -->
 
+## transaction-text
+
+> **The ‘transaction-text’ parameter specifies the text displayed when a new transaction occurs.**
```

Comment:
Let's clarify that it's displayed when a new transaction is received that doesn't have the correct amount & OP_RETURN code.

On BCH we just check that it's the correct amount as OP_RETURN is not really supported well on the wallet side.